packages/clay-nav/README.mdx

```diff
@@ -1,17 +1,53 @@
-# clay-nav
+---
+title: 'Navigation'
```

Comment:
Lets leave the name as `Nav` since that is the name of our component. Navigation could be all sorts of components